Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -448,19 +449,34 @@ void ProjectData::parseMedia(
          media_config->getConfigSubtreeList("medium"))
     {
         //! \ogs_file_attr{prj__media__medium__id}
-        auto const material_id = medium_config.getConfigAttribute<int>("id", 0);
+        auto const material_id_string =
+            medium_config.getConfigAttribute<std::string>("id", "0");
+        std::vector<std::string> material_ids;
+        boost::split(material_ids, material_id_string, boost::is_any_of(","));
 
-        if (_media.find(material_id) != _media.end())
+        for (auto const& m_id : material_ids)
         {
-            OGS_FATAL(
-                "Multiple media were specified for the same material id %d. "
-                "Keep in mind, that if no material id is specified, it is "
-                "assumed to be 0 by default.",
-                material_id);
-        }
+            if (m_id.find_first_not_of("0123456789") != std::string::npos)
```

Comment:
Minor; maybe `boost::algorithm::is_digit` is more idiomatic.

Reply:
changed it to the boost solution